Documentation/CrossPlatform.md

```diff
@@ -3,36 +3,38 @@
+Building the dockerfile referenced above is the simplest way to get a fresh build of ImageMagick itself. The dockerfile includes configuration and compilation 
+steps for ImageMagick and it's dependencies, and the Magick.NET.CrossPlatform project will link against that build. To build against a different version of 
+ImageMagick or a particular image library, simply edit the dockerfile and change the relevant git checkout to reference the tag or commit of your choice (or 
+just let it get latest). You can also use the `apt-get install` command in the dockerfile to control which formats and compression schemes are supported. 
 
-- Clone the Magick.NET repository from `https://github.com/dlemstra/Magick.NET.git`.
-- Switch to the tag that has the same version as the version of Magick.NET (for example `git checkout tags/1.2.3.4`).
-- Run `ImageMagick/Source/Checkout.sh` that will clone the ImageMagick repository.
-- An optional step is `sudo apt-get build-dep imagemagick` that will install all the dependencies.
-- Go to the folder `ImageMagick/Source/ImageMagick/ImageMagick` and run one of the following commands (depening on the quantum)
-    - Q8: `./configure --with-magick-plus-plus=no --with-quantum-depth=8 --enable-hdri=no`
-    - Q16: `./configure --with-magick-plus-plus=no --with-quantum-depth=16 --enable-hdri=no`
-    - Q16-HDRI: `./configure --with-magick-plus-plus=no --with-quantum-depth=16`
-- The next step is `make install` or `sudo make install`.
+For example, to build against IM 7.0.7-20, go to `Source/Magick.NET.CrossPlatform/ubuntu.16.04` and make sure the dockerfile includes this:
+
+```
+RUN cd ~/ImageMagick; git checkout tags/7.0.7-20;
+```
+
+Then rerun `docker build` and start a fresh container from the result. This will create an Ubuntu 16.04 container with a fresh copy of ImageMagick and various 
+image libraries, ready to support building Magick.NET.Native from Visual Studio or the BuildCrossPlatform script.
 
 ## Extra requirements
 
-Magick.NET is also linked again `libjpeg.so.8` (used by the `JpegOptimizer`). Not all Linux platforms have this available by
-default which means it might be required to build [libjpeg-turbo](https://www.libjpeg-turbo.org/) from source. On Windows the following version of
-`libjpeg-turbo` is used: https://github.com/ImageMagick/jpeg-turbo.
+By default, Magick.NET has support built in for JPEG, TIFF, PNG, and WebP file formats. Lossless compression support includes zlib, lzma, and bzip2. There
+are no external dependencies beyond basic system libraries and glibc6/libm, which should be present by default on most Linux distros. 
 
-## Using Magick.NET on Linux
+Distributions based on musl libc such as Alpine will not have a glibc available, and will not work with Magick.NET out of the box. Compatibility layers for 
+glibc on musl-libc systems exist, but have not been tested successfully with Magick.NET. YMMV.
+
+Windows-based systems should not need any extra components or configuration.
```

Comment:
Not sure if we need the Windows comment here because we are talking about a linux people.

p.s. thanks for also changing the documentation.